shared/actions/chat2/team-building.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import logger from '../../logger'
+import * as TeamBuildingTypes from '../../constants/types/team-building'
+import * as TeamBuildingGen from '../team-building-gen'
+import * as Chat2Gen from '../chat2-gen'
+import * as RouteTreeGen from '../route-tree-gen'
+import * as Saga from '../../util/saga'
+import * as RPCTypes from '../../constants/types/rpc-gen'
+import {trim} from 'lodash-es'
+import {type TypedState} from '../../constants/reducer'
+
+type RawSearchResult = {
+  score: number,
+  keybase: ?{
+    username: string,
+    uid: string,
+    picture_url: ?string,
```

Comment:
lets start just typing these are string and set '' , i find this simplifies the typing down the road